Library/Formula/ddar.rb

```diff
@@ -9,7 +9,7 @@ class Ddar < Formula
 
   depends_on "xmltoman" => :build
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on "protobuf"
```

Comment:
Oh, yep, that's it: on the machine where this works, I have a `~/Library/Python/2.7/lib/python/site-packages/homebrew.pth`, created per the Caveats you see when you install a formula with Python modules using the system Python. (e.g. `PATH=/usr/bin:$PATH; brew install protobuf`.) On the machine where it doesn't work, I have no such file.

What's the word on `#!` python paths for Homebrew-installed programs? I don't see anything in [Homebrew and Python](). The hardcoded `#!/usr/bin/python` makes `ddar` run against system python regardless of your `$PATH` settings or how it was built. `ranger.rb` seems to handle a similar issue with `s.gsub! "#!/usr/bin/python", "#!#{PythonRequirement.new.which_python}"`; maybe we should do that here too.